Tweak: Document - JS API/Docs/Components, Explanation of defaultUtils. [ED-4162] (#15610)
elementor/elementor: docs/core/common/assets/js/api/core/components.md

    
    export default class Component;
    ```
* Placement of component methods, and usage of `defaultUtils`
	* Assuming you method which used from the commands across your component *namespace*, those methods can be placed as methods at the components level, see: `createAlert`.

> 	* If the method is used outside the component you should use `defaultUtils`, see at code below.

Reviewer: ```suggestion
	* If the method is being used outside of the component, you should use `defaultUtils`. See the example below:
```

Reviewer 2: I didn't understand why. there are more conventional ways to separate between public/private methods.

Author: > I didn't understand why. there are more conventional ways to separate between public/private methods.

I didn't understand what you mean, the difference in placement of method in component vs defaultUtils.

You can create methods in component and use them only if they being called from the namespace scope.

If the method is used outside the namespace scope, we can use 'defaultUtils'.

If you have alternative or explanations, please suggest.

Reviewer 2: If a utility does not belong to the component it should be written as a utilility in the utils folder.

Reviewer 2: Public utils should be exported via ES6 `export`, otherwise it should be a native method. + remove the `defaultUtils`
